gpii/node_modules/flowManager/src/FlowManager.js

```diff
@@ -131,45 +123,32 @@ fluid.defaults("gpii.flowManager.save", {
 });
 
 
-fluid.defaults("gpii.flowManager.io", {
-    gradeNames: ["autoInit", "fluid.littleComponent"],
-    handlers: {
+fluid.defaults("gpii.flowManager.ws", {
+    requestHandlers: {
         userUpdate: {
             route: "/update",
-            type: "io"
-        },
-        userLogin: {
-        },
-        userLogout: {
-        },
-        userSavePost: {
-        },
-        userSavePut: {
-        },
-        getUserToken: {
-        },
-        userLogonStateChange: {
+            type: "gpii.flowManager.userUpdate.handler"
         },
         browserChannel: {
             route: "/browserChannel",
-            type: "io"
+            type: "gpii.flowManager.browserChannel.handler"
         }
     },
     components: {
-        webSocketsComponent: {
-            type: "gpii.settingsHandlers.webSocketsComponent"
+        webSocketsSettingsHandler: {
+            type: "gpii.settingsHandlers.webSockets.component"
```

Comment:
Please correct the use of the old component name ```gpii.settingsHandlers.webSocketsComponent``` and the js file name ```WebSocketsComponent.js``` in the documentation for BrowserChannel: https://github.com/amb26/universal/blob/GPII-1318/documentation/BrowserChannel.md, line 7 & 17.